docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1,86 +1,87 @@
+There are some properties that could be shared among different Azure services, for example using the same service principal to access Azure Cosmos DB and Azure Event Hubs. Spring Cloud Azure 4.0 allows developers to define properties that apply to all Azure SDKs with the prefix `spring.cloud.azure`.
 
 .Global properties
 [cols="1,3", options="header"]
 |===
 |Property  |Description
 
-|*spring.cloud.azure*.client
+|spring.cloud.azure.*client*
 |To configure the transport clients apply to all Azure SDKs by default. 
 
-|*spring.cloud.azure*.credential
+|spring.cloud.azure.*credential*
 |To configure how to authenticate with Azure Active Directory for all Azure SDKs by default. 
 
-|*spring.cloud.azure*.profile
+|spring.cloud.azure.*profile*
 |To configure the Azure cloud environment for all Azure SDKs by default. 
 
-|*spring.cloud.azure*.proxy
+|spring.cloud.azure.*proxy*
 |To configure the proxy options apply to all Azure SDK clients by default. 
 
-|*spring.cloud.azure*.retry
+|spring.cloud.azure.*retry*
 |To configure the retry options apply to all Azure SDK clients by default. 
 |===
 
 
 NOTE: Properties configured under each Azure service will override the global configurations.
 
-The configuration properties' prefixes have been unified to `spring.cloud.azure` namespace since Spring Cloud Azure 4.0, it will make configuration properties more consistent and more intuitive. Here's a quick review of the prefixes of supported Azure services.
+Spring Cloud Azure unifies configuration properties' prefixes to `spring.cloud.azure` since 4.0, which will make configuration properties more consistent and more intuitive. Here's a quick review of the prefixes of each supported Azure service.
 
 .Service specific properties
 [cols="<50,<50", options="header"]
 |===
 |Azure Service |Configuration Property Prefix 
 
 |Azure App Configuration 
-|*spring.cloud.azure*.appconfiguration
+|spring.cloud.azure.*appconfiguration*
 
 |Azure Cosmos DB 
-|*spring.cloud.azure*.cosmos
+|spring.cloud.azure.*cosmos*
 
 |Azure Event Hubs 
-|*spring.cloud.azure*.eventhubs
+|spring.cloud.azure.*eventhubs*
 
 |Azure Key Vault Certificate 
-|*spring.cloud.azure*.keyvault.certificate
+|spring.cloud.azure.*keyvault.certificate*
 
 |Azure Key Vault Secret 
-|*spring.cloud.azure*.keyvault.secret
+|spring.cloud.azure.*keyvault.secret*
 
 |Azure Service Bus 
-|*spring.cloud.azure*.servicebus
+|spring.cloud.azure.*servicebus*
 
-|Azure Storage Blob 
-|*spring.cloud.azure*.storage.blob
+|Azure Blob Storage
+|spring.cloud.azure.*storage.blob*
 
 |Azure Storage File Share 
-|*spring.cloud.azure*.storage.fileshare
+|spring.cloud.azure.*storage.fileshare*
 
-|Azure Storage Queue 
-|*spring.cloud.azure*.storage.queue
+|Azure Queue Storage
+|spring.cloud.azure.*storage.queue*
```

Comment:
Should we keep using Azure Storage XX? because Azure Storage XX is used elsewhere.

Reply:
good point, @backwind1233 this is what i'd like to discuss, seems like the Storage Blob service is called Azure Blob Storage?

Reply:
In my mind

Azure Storage Blob > Azure  Blob > Azure Blob Storage


https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction

Reply:
@backwind1233 So we stick to Azure Storage Blob here?